lib/pr/fetcher.rb

```diff
@@ -0,0 +1,145 @@
+class PR
+  class Fetcher
+    HTTP = GraphQL::Client::HTTP.new("https://api.github.com/graphql") do
+      def headers(context)
+        unless token = context[:access_token] || ENV['GH_TOKEN']
+          fail "Missing GitHub access token"
+        end
+
+        {
+          "Authorization" => "Bearer #{ENV['GH_TOKEN']}"
```

Comment:
You're setting a `token` variable a few lines before this; should you be using that here?

Reply:
Good point. I missed that one.